pyanaconda/argument_parsing.py

```diff
@@ -544,20 +543,20 @@ def __call__(self, parser, namespace, values, option_string=None):
             if values:
                 options = values.split(",")
             if "all" in options:
-                flags_instance.nosave_input_ks = True
-                flags_instance.nosave_output_ks = True
-                flags_instance.nosave_logs = True
+                setattr(namespace, "save_input_kickstart", False)
```

Comment:
I think you can do `namespace.save_input_kickstart = False`. Also, I suggest to initialize these values to `None` at the beginning of the method.